src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -135,13 +139,13 @@ internal override MemberSemanticModel GetMemberModel(SyntaxNode node)
             return IsInTree(node) ? this : null;
         }
 
-        protected NullableWalker.SnapshotManager SnapshotManager
+        // We hide this property, as EnsureRootBoundForNullabilityIfNecessary can cause
```

Comment:
>We hide this property, as [](start = 11, length = 25)

This is a little confusing since this member is not a property. Perhaps "Not a property because ...".